lib/ansible/modules/user.py

```diff
@@ -2510,6 +2510,14 @@ def create_user(self, command_name='dscl'):
         if rc != 0:
             self.module.fail_json(msg='Cannot create user "%s".' % self.name, err=err, out=out, rc=rc)
 
+        # Make the Gecos (alias display name) default to username
+        if self.comment is None:
+            self.comment = self.name
+
+        # Make user group default to 'staff'
+        if self.group is None:
+            self.group = 'staff'
+
```

Comment:
These defaults should probably be mentioned in the documentation for these module arguments.